projects/documentation/content/guides/configuring-openwc.md

```diff
@@ -0,0 +1,108 @@
+---
+layout: guide.njk
+title: 'Configuring your @open-wc project'
+displayName: Configuring your project
+slug: configuring-openwc
+---
+
+This guide expounds upon configuration specifics you may want to adopt in your `@open-wc` project for a smoother development experience that matches the specifications in Spectrum Web Components.
+
+## Building your project
+
+In your terminal, run the following command:
+
+```bash
+npm init @open-wc
+```
+
+Once you do that, `@open-wc` will give you a few configuration options. You'll want to set up your project as a Web Component and allow it to use typescript. When it asks you what features to add, say "yes" to all of them--we'll explain how to set them up to be more compatible with SWC in this guide. SWC uses `yarn` to manage its package dependencies, so you can go ahead and choose that one, as well.
+
+## Setting up your .gitignore
+
+The first thing to check is that your `.gitignore` file isn't set up to ignore changes to your root directory. By default, the `.gitignore` in `@open-wc` starts each of its ignored directories with a `/`. You can go ahead and replace the contents of the file with what's below, which is the same file list which omits the forward slashes.
+
+```bash
+## editors
+.idea
+.vscode
+
+## system files
+.DS_Store
+
+## npm
+node_modules/
+npm-debug.log
+
+## testing
+coverage/
+
+## temp folders
+.tmp/
+
+# build
+_site/
+dist/
+out-tsc/
+
+storybook-static
+custom-elements.json
+```
+
+## Setting up your linter
+
+While both SWC and `@open-wc` use `es-lint` and `prettier`, there are some differences in their configurations that you may want to adjust if you plan on contributing your code to Spectrum Web Components later down the line.
+
+The biggest one is the indentation. In your `.editorconfig` file, clear the file contents and replace it with:
+
+```yaml
+root = true
+
+[*]
+end_of_line = lf
+indent_style = space
+indent_size = 4
+charset = utf-8
+```
+
+Now, in your root directory, create a new file called `.prettierrc.yaml`. In it, add the following:
+
+```yaml
+printWidth: 80
+tabWidth: 4
+semi: true
+singleQuote: true
+trailingComma: es5
+bracketSpacing: true
+arrowParens: always
+htmlWhitespaceSensitivity: ignore
+```
+
+Also create a file in your root called `.prettierignore`, and in it, put:
+
+```
+node_modules
```

Comment:
This seems like it's the default... https://prettier.io/docs/en/ignore.html#ignoring-files-prettierignore should we leave this out?

Reply:
There wasn't a prettierignore in the generated project, so I assumed I should go all the way with it and add one. Good to see the default though. I can take it out.